integration-tests/src/test/resources/domain-home-on-pv/create-domain.py

```diff
@@ -116,6 +116,12 @@ def getEnvVar(var):
   cmo.setCluster(cl)
   print('Done setting attributes for Server Template: %s' % templateName);
 
+  templateChannelName = cluster_name + "-NAP"
+  create(templateChannelName, 'NetworkAccessPoint')
+  cd('NetworkAccessPoints/%s' % templateChannelName)
+  set('PublicPort', server_port + 10)
+  set('ListenAddress', '%s-%s${id}' % (domain_uid, managed_server_name_base_svc))
+  set('ListenPort', server_port + 10)
```

Comment:
The same set of changes also need to be made for configured clusters, otherwise, once configured cluster support is tested again, the test will fail...

Reply:
Yes. I that case, should I add the NAP to each individual server?

Reply:
@tbarnes-us Is it ok if I file a jira for this? Or should I simply check for the NAP in the lifecycle checks if this is a dynamic cluster?